Example/Custom Integration/CardExampleViewController.m

```diff
@@ -0,0 +1,197 @@
+//
+//  CardExampleViewController.m
+//  Custom Integration (ObjC)
+//
+//  Created by Daniel Jackson on 7/5/18.
+//  Copyright © 2018 Stripe. All rights reserved.
+//
+
+@import Stripe;
+
+#import "CardExampleViewController.h"
+#import "BrowseExamplesViewController.h"
+
+/**
+ This example demonstrates using PaymentIntents to accept card payments verified using 3D Secure.
+
+ 1. Collect user's card information via `STPPaymentCardTextField`
+ 2. Create a `PaymentIntent` on our backend (this can happen concurrently with #1)
+ 3. Confirm PaymentIntent using the `STPPaymentMethodParams` for the user's card information.
+ 4. If the user needs to go through the 3D Secure authentication flow, use `STPRedirectContext` to do so.
+ 5. When user returns to the app, or finishes the SafariVC redirect flow, `STPRedirectContext` notifies via callback
+
+ See the documentation at https://stripe.com/docs/payments/dynamic-authentication for more information
+ on using PaymentIntents for dynamic authentication.
+ */
+@interface CardExampleViewController () <STPPaymentCardTextFieldDelegate>
```

Comment:
Let's call this `CardAutomaticConfirmationViewController` in line with the terminology here https://stripe.com/docs/payments/payment-intents/quickstart#flow-automatic-confirmation

Reply:
👍